projects/packages/forms/src/modules/form/view.js

```diff
@@ -132,7 +132,7 @@ const { state } = store( NAMESPACE, {
 			if ( context?.maxSteps && context.maxSteps > 0 ) {
 				return false;
 			}
-			return ! Object.values( context.fields ).some( field => field.value !== '' );
+			return ! Object.values( context.fields ).some( field => field.value.trim() !== '' );
```

Comment:
Are we sure all values are strings there? should we `field?.value?.trim?.()`?

Reply:
Yeah. Checkboxes are causing issues here. Fixing.